qiskit/transpiler/passes/scheduling/calibration_creators.py

```diff
@@ -269,27 +278,23 @@ class RZXCalibrationBuilderNoEcho(RZXCalibrationBuilder):
     """
     Creates calibrations for RZXGate(theta) by stretching and compressing
     Gaussian square pulses in the CX gate.
-
-    The ``RZXCalibrationBuilderNoEcho`` is a variation of the
-    :class:`~qiskit.transpiler.passes.RZXCalibrationBuilder` pass
-    that creates calibrations for the cross-resonance pulses without inserting
+    The RZXCalibrationBuilderNoEcho is a variation of the RZXCalibrationBuilder
+    as it creates calibrations for the cross-resonance pulses without inserting
```

Comment:
Can we keep the sphinx `:class: ... `formatting for the docs?

Reply:
See d600452.